ConcurrentModificationExceptionResolved/src/com/webonise/concurrentmodification/ConcurrentModification.java

```diff
@@ -0,0 +1,31 @@
+package com.webonise.concurrentmodification;
+
+import java.util.Iterator;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+/**
+ * Created by webonise on 19/9/16.
+ */
+public class ConcurrentModification {
+    public static void main(String args[]) {
+        //To avoid  ConcurrentModificationException we will useCopyOnWriteArrayList
+        List<String> myList = new CopyOnWriteArrayList<>();
```

Comment:
To get better understanding of `ConcurrentModificationException  ` and `Iterator ` better use `ArrayList`, and while iterating over the list modify the list to produce `ConcurrentModificationException`. And you can make use of `Iterator `to avoid `ConcurrentModificationException`.